huaweicloud/resource_huaweicloud_dns_zone_v2.go

```diff
@@ -47,6 +47,7 @@ func ResourceDNSZoneV2() *schema.Resource {
 			"email": {
 				Type:     schema.TypeString,
 				Optional: true,
+				Computed: true,
```

Comment:
seems we only add `Computed` if it has a default value

Reply:
if it has a default value, suggest to update the basic test case with no email specified.

Reply:
done.